test/spinxsk/spinxsk.c

```diff
@@ -299,6 +299,17 @@ RandUlong(
     return r;
 }
 
+VOID
+FillRandomBuffer(
```

Comment:
nit/fyi: our team's preferred style is to try to put `_Out_ Foo, _In_ Bar` as the parameter order. there's so much inconsistency i wouldn't bother fixing this here, though

Reply:
I can move to whatever becomes the standard. I tend to follow our internal guidelines (https://www.osgwiki.com/wiki/Cpp_Core_Guidelines#OUT.21:_Prefer_to_order_output_parameters_last) when they make sense to C.

Reply:
We tend to follow traditional C standard library and NT kernel style, which, for example, `memset` and `RtlFillMemory` both have the `_Out_` parameter first. I don't think we have the `_Out_` rule written down, partly because the most general rule of "make the code look like nearby code" tends to overrule any more specific rule in this case.